packages/@okta/vuepress-site/docs/reference/api/authn/index.md

```diff
@@ -881,6 +881,7 @@ Include the `X-Device-Fingerprint` header to supply a device fingerprint. The `X
 
 * If the new or unknown device email notification is enabled, an email is sent to the user if the device fingerprint sent in the `X-Device-Fingerprint` header isn't associated with a previously successful user sign in. For more information about this feature, see the [General Security documentation]https://help.okta.com/en/prod/okta_help_CSH.htm#ext_Security_General.
 * If you have the security behavior detection feature enabled and you have a new device behavior configured in a policy rule, a new device is detected if the device fingerprint sent in the `X-Device-Fingerprint` header isn't associated with a previously successful user sign in. See [New Device Behavior Detection](https://help.okta.com/en/prod/okta_help_CSH.htm#ext_proc_security_behavior_detection).
+> **Note:** The use of the `X-Device-Fingerprint` header for new device security behavior detection is deprecated. Starting April 12 2021, we are going to enable [improvements to the new device security behavior](https://help.okta.com/en/prod/Content/Topics/Security/improved-new-device-behavior-detection.htm) for all the existing tenants. Once the improvements are rolled out, new device security behavior would only rely on the `deviceToken` in the [Context Object](#context-object) and would not rely on the `X-Device-Fingerprint` header.
```

Comment:
We would have to discuss with PM on the date we want to include in this doc. I chose April 12th as a place holder for now.

Reply:
I am skeptical of including a date at all. We have already communicated a number of deadlines and we have missed all of them so far